README.rst

```diff
@@ -178,62 +178,86 @@ GetComponentImages example:
     image = Image.open('/usr/src/tesseract/testing/phototest.tif')
     with PyTessBaseAPI() as api:
         api.SetImage(image)
-        boxes = api.GetComponentImages(RIL.TEXTLINE, True)
-        print('Found {} textline image components.'.format(len(boxes)))
-        for i, (im, box, _, _) in enumerate(boxes):
-            # im is a PIL image object
-            # box is a dict with x, y, w and h keys
-            api.SetRectangle(box['x'], box['y'], box['w'], box['h'])
-            ocrResult = api.GetUTF8Text()
```

Comment:
Note: it's not enirely wrong to use the API this way, because `GetComponentImages` makes copies of the segment images and bboxes, so it does not hurt that `SetRectangle` invalidates the layout analysis results. But it is still not useful to loop that way if the ultimate goal is the text – you would rather look into the iterator directly for the text. Also, in this formulation, you would have needed to at least set the PSM to line level for a decent OCR result.